docs/rpc@usage/nameserver-daemon.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+
+### BEGIN INIT INFO
+# Provides:          myservice
+# Required-Start:    $remote_fs $syslog
+# Required-Stop:     $remote_fs $syslog
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: Put a short description of the service here
+# Description:       Put a long description of the service here
+### END INIT INFO
+
+# Change the next 3 lines to suit where you install your script and what you want to call it
+DIR=/home/user/ims
+DAEMON=nameserver.py
+DAEMON_NAME=nameserver
+
+# Add any command line options for your daemon here
+DAEMON_OPTS=""
```

Comment:
Can we have a single config file from we can read all the parameters and pass on to subsequent python scripts